controllers/front/OrderController.php

```diff
@@ -237,6 +237,46 @@ public function displayAjaxselectDeliveryOption()
         ]));
     }
 
+    public function displayAjaxcheckCustomerInformation(): void
+    {
+        $email = Tools::getValue('email');
+        $isEmailValid = Validate::isEmail($email);
+        $isGuestAllowed = (bool) Configuration::get('PS_GUEST_CHECKOUT_ENABLED');
+        $customerExists = false;
+        $responseData = [];
+        $alert = [];
+
+        if ($isEmailValid) {
+            $customerExists = Customer::customerExists($email, false, true);
+        }
+
+        if ($customerExists) {
+            $alert['type'] = $isGuestAllowed ? 'info' : 'danger';
+            $alert['message'] = $isGuestAllowed ?
+                $this->translator->trans(
+                    'You already have an account, don\'t hesitate to log in!',
```

Comment:
```suggestion
                    'You already have an account, don\'t hesitate to sign in!',
```

Please use "Sign in" for more consistency. :-)
 
![image](https://user-images.githubusercontent.com/70583503/128192981-86c0e607-226a-48e5-80b3-98c2f182a877.png)